src/user.rs

```diff
@@ -0,0 +1,46 @@
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+
+use rustc_serialize::json;
+
+use project_types::Project;
+use Error;
+
+
+pub fn new_user<P: AsRef<Path>>(project: String, public: P, name: String) -> Result<(), Error> {
+
+	// Get public key from file
+	let pub_key = try!(file_as_string(public));
+
+	// Get existing Protonfile
+    let mut path = PathBuf::from(Path::new(&project));
+    path.push("Protonfile.json");
+    let protonfile = try!(file_as_string(&path));
+
+    // Load Project from existing file
```

Comment:
Make this its own function `fn read_protonfile() -> Project`, and move it into a Utils module. Definitely going to be needed elsewhere.